documents/src/pages/elements/notification.md

```diff
@@ -32,7 +32,7 @@ The notification bar is used to show informative content when something happens
 These methods work in a similar way to those provided by `window.console`.
 
 ```javascript
-import { info, confirm, warn, error } from '@refinitiv-ui/notification/helpers';
+import { info, confirm, warn, error } from '@refinitiv-ui/elements/notification';
```

Comment:
I had tried to implement this import with example code for notification, and I seems not to work for me.